localisation/00_zuwc_united_localisation.csv

```diff
@@ -2848,6 +2848,14 @@ GetMonstrousUndeadNameDesc_Abomination;a horrific creation that is sewn together
 EVTDESC_WCPOD_5013_wickerman;Your days in a living body come to an end. Thros offered  you a new life, giving you the opportunity to tie your soul to the wicker revenant, removing the limitations and weaknesses of flesh.;;Eure Tage in einem lebenden K�rper gehen zu Ende. Thros hat Euch ein neues Leben geschenkt und Euch die M�glichkeit gegeben, Eure Seele an den Weidenwiederg�nger zu binden und die Begrenzungen und Schw�chen des Fleisches zu �berwinden.;;;;;;;;;;;x
 EVTDESC_WCPOD_5013_wicker_golem;Your days in a living body come to an end. Thros offered you a new life, giving you the opportunity to tie your soul to the wicker golem, greatly improving your physical strength and turning you into a fearsome machine of war.;;Eure Tage in einem lebenden K�rper gehen zu Ende. Thros schenkte Euch ein neues Leben und gab Euch die M�glichkeit, Eure Seele an den Weidengolem zu binden, was Eure K�rperkraft stark erh�hte und Euch in eine furchteinfl��ende Kriegsmaschine verwandelte.;;;;;;;;;;;x
+wastewander_opinion;[wastewander.GetName] Opinion;;[wastewander.GetName] Meinung;;;;;;;;;;;x
+has_castle_or_city_tooltip;Has a �YCastle�! or �YCity�!;;Besitzt eine �YBurg�! oder �YStadt�!;;;;;;;;;;;x
+end_of_scourge;Blood of [This.GetFirstNameWithNick];;Blut von [This.GetFirstNameWithNick];;;;;;;;;;;x
+end_of_scourge_desc;[This.GetFirstNameWithNick] was the one who led [This.GetHisHer] fearless armies against undead hordes of the Scourge and emerged from this battle as a winner.;;[This.GetFirstNameWithNick] war [This.Gerderdie]jenige, [This.Getderdie] [This.GetHisHer]e furchtlose Armeen gegen die untoten Horden der Gei�el f�hrte und aus dieser Schlacht als Sieger[This.GetinFemale] hervorging.;;;;;;;;;;;x
+end_of_legion;Blood of [This.GetFirstNameWithNick];;Blut von [This.GetFirstNameWithNick];;;;;;;;;;;x
+end_of_legion_desc;[This.GetFirstNameWithNick] was the one who led [This.GetHisHer] fearless armies against demonic hordes of the Burning Legion and emerged from this battle as a winner.;;[This.GetFirstNameWithNick] war [This.Gerderdie]jenige, [This.Getderdie] [This.GetHisHer]e furchtlose Armeen gegen die d�monischen Horden der brennenden Legion f�hrte und aus dieser Schlacht als Sieger[This.GetinFemale] hervorging.;;;;;;;;;;;x
+end_of_black_empire;Blood of [This.GetFirstNameWithNick];;Blut von [This.GetFirstNameWithNick];;;;;;;;;;;x
+end_of_black_empire_desc;[This.GetFirstNameWithNick] was the one who led [This.GetHisHer] fearless armies against faceless hordes of the Black Empire and emerged from this battle as a winner.;;[This.GetFirstNameWithNick] war [This.Gerderdie]jenige, [This.Getderdie] [This.GetHisHer]e furchtlose Armeen gegen die gesichtslosen Horden des schwarzen Imperiums f�hrte und aus dieser Schlacht als Sieger[This.GetinFemale] hervorging.;;;;;;;;;;;x
```

Comment:
Remove all text prior to "led"

Reply:
Done in 91c2397093aba3fa5e7c13cfb7e16f9da2f475e5.